x-pack/plugins/ingest_manager/common/services/datasource_to_agent_datasource.test.ts

```diff
@@ -26,7 +26,7 @@ describe('Ingest Manager - storedDatasourceToAgentDatasource', () => {
   const mockInput: DatasourceInput = {
     type: 'test-logs',
     enabled: true,
-    config: {
+    pkg_variables: {
```

Comment:
Why do you prefix these by `pkg_`?

Reply:
it's used only to populate the stream template from pkg, I think it make it more explicit